_includes/pbc/all/install-features/202410.0/install-the-payments-feature.md

```diff
@@ -32,7 +32,7 @@ composer require "spryker-feature/payments:{{page.version}}" "spryker/checkout-r
 {% info_block warningBox "Verification" %}
 Make sure that the following modules have been installed:
 
-| MODULE                    | EXPECTED DIRECTORY                               | REQUIRED ONLY FOR ACP PAYMENTS |
+| MODULE                    | EXPECTED DIRECTORY                               |  REQUIRED ONLY FOR ACP PAYMENTS |
```

Comment:
Do you have an idea what we're trying to explain by the `REQUIRED ONLY FOR ACP PAYMENTS` column in this table?

Reply:
I'd say that this is just a note that for SCOS payment this is not needed and only has an effect when used with ACP Payment Apps.